src/routes/[...lang=locale]/anime/+page.svelte

```diff
@@ -139,43 +139,43 @@
 			banner: "/images/mock/banner/one_piece.avif",
 			title: "One Piece",
 			ep_number: 20000,
-			timestamp: "1 hour ago", // TODO: format time
+			timestamp: "1 hour ago" // TODO: format time
 		},
 		{
 			id: 2,
 			cover: "/images/mock/cover/jjk.webp",
 			banner: "/images/mock/banner/jjk.jpg",
 			title: "Jujutsu Kaisen season 2",
 			ep_number: 5,
-			timestamp: "2 hour ago", // TODO: format time
+			timestamp: "2 hour ago" // TODO: format time
 		},
 		{
 			id: 3,
 			cover: "/images/mock/cover/demon_slayer_training.webp",
 			banner: "/images/mock/banner/demon_slayer_training.avif",
 			title: "Demon Slayer Hashira Training Arc",
 			ep_number: 2,
-			timestamp: "2 hour ago", // TODO: format time
+			timestamp: "2 hour ago" // TODO: format time
 		},
 		{
 			id: 4,
 			cover: "/images/mock/cover/kaiju_no_8.jpg",
 			banner: "/images/mock/banner/kaiju_no_8.webp",
 			title: "Kaiju no.8",
 			ep_number: 1,
-			timestamp: "3 hour ago", // TODO: format time
+			timestamp: "3 hour ago" // TODO: format time
 		}
 	];
 	// state for latest_episodes color palette
 	const latest_episodes_color_palette_mapping: {
 		[key: number]: [number, number, number][];
 	} = $state({});
+	const latest_episodes_loaded_mapping: boolean[] = $state(
+		new Array(latest_episodes.length).fill(false)
+	);
```

Comment:
This 2  needs to be merged, or we should have a better structure than this shit

Reply:
Resolved on https://github.com/baseplate-admin/coreproject-v4-ui/pull/77/commits/88f5310849a03bf81c48b69abd12e50153f6c5fb